schema/TRANSFER.json

```diff
@@ -306,7 +306,7 @@
             "description": "",
             "type": "number",
             "minimum": 0,
-            "maximum": 999999999999,
+            "maximum": 999999999,
```

Comment:
@Elaine-Krauss-TCG Good catch! But I believe the correct number is: 999999999.99
See the other schemas for verification
Same below, too, of course

... looks like PAC_RECOUNT_RECEIPT and SchB also has an incorrect number formats that need correcting. Would you do that in the PR, too?

Reply:
Got it, thanks!